Rubberduck.Core/UI/Settings/SettingsForm.cs

```diff
@@ -19,7 +22,7 @@ public SettingsForm(IGeneralConfigService configService, IOperatingSystem operat
                 config,
                 new SettingsView
                 {
-                    Control = new GeneralSettings(new GeneralSettingsViewModel(config, operatingSystem)),
+                    Control = new GeneralSettings(new GeneralSettingsViewModel(config, operatingSystem, new ReadOnlyCollection<Type>(new List<Type>()))),
```

Comment:
See above

Reply:
actually, we can drop the ReadOnly specification here, because the ViewModel takes an IEnumerable